Demo/Demo/SwiftUIComponents/CardPaymentViews/CardOrderCompletionResultView.swift

```diff
@@ -35,6 +35,24 @@ struct CardOrderCompletionResultView: View {
             LeadingText("\(orderResponse.id)")
             LeadingText("Status", weight: .bold)
             LeadingText("\(orderResponse.status)")
+            if let lastDigits = orderResponse.paymentSource?.card.lastDigits {
```

Comment:
Since this seems to be the same logic below can we can maybe extract this out into a method where we pass in the `orderResponse.paymentSource`?

Reply:
Not sure what you mean? Do you mean checking for values in paymentSource?

Reply:
Addressed 8b7702c71cdef997347e2ea250ff88f6bd107d57

Reply:
Nice! 👏